Cargo.lock

```diff
@@ -417,7 +417,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 
 [[package]]
 name = "xargo"
-version = "0.3.12"
+version = "0.3.13"
```

Comment:
This diff seems surprising?

Reply:
Surprising, but necessary. Forgotten in 590bdd23ff92fdc76b7e3fd57ddc6cd6785ce388

Reply:
Ah, makes sense.